tests/conftest.py

```diff
@@ -74,3 +74,30 @@ def build_jupyter_book(
     check_subproc_run_for_nonzero(completed_proc=result)
 
     return jupyter_book_test_target
+
+
+from data.almanack.entropy.add_data import add_data
+
+
+@pytest.fixture(scope="session")
+def repository_paths():
+    """
+    Fixture to call add_Data.py, create the repositores, then delete them
+    """
+    # Compute the base path relative to this location
+    base_path = (
```

Comment:
Consider the use of the built-in [`tmp_path`](https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html) fixture which can provide a temporary directory for use with tests. The nice part about these is that they are automatically removed once processing completes. Using this might also involve adding an additional parameter for directory specification within `add_data()`.